lib/fluent/supervisor.rb

```diff
@@ -568,7 +568,7 @@ def show_plugin_config
 
     def supervise
       # Make dumpable conf, which is set corresponding_proxies for all elements in all worker sections
-      dry_run
```

Comment:
Supervisor process may have to change privilege to run `configure`. So it seems better that supervisor call `change_privilege` and worker process doesn't call `change_privilege` here:  https://github.com/fluent/fluentd/blob/5a0e14776b737ade4ca5a002bfbd0287d219dcb3/lib/fluent/supervisor.rb#L520